controller/src/beerocks/master/tasks/bml_task.cpp

```diff
@@ -193,6 +193,32 @@ void bml_task::handle_event(int event_type, void *obj)
         }
         break;
     }
+    case REGISTER_TO_TOPOLOGY_UPDATES: {
+        if (obj) {
+            auto event_obj = (listener_general_register_unregister_event *)obj;
+            TASK_LOG(DEBUG) << "REGISTER_TO_TOPOLOGY_UPDATES event was received";
+            database.add_bml_socket(event_obj->sd);
+            if (!database.set_bml_topology_update_enable(event_obj->sd, true)) {
+                TASK_LOG(DEBUG) << "fail in topology_updates registration";
+            }
+            state = LISTENING;
+        }
+        break;
```

Comment:
```cpp
if (!obj) {
    // Handle error
}

// Rest of the code

Reply:
I prefer not to add breaks in the middle of the code